cg/cli/delete/observations.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import click
+from cg.apps.loqus import LoqusdbAPI
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group("delete")
+def delete_cmd():
+    """Delete variants associated to a case in LoqusDB."""
+    pass
+
+
+@delete_cmd.command()
+@click.option("-c", "--case_id", help="internal case id")
+@click.pass_obj
+def observations(context: CGConfig, case_id: str):
+    """Delete case from LoqusDB and reset loqusdb_id in statusDB."""
+
+    status_db: Store = context.status_db
+    loqusdb_api: LoqusdbAPI = context.loqusdb_api
+
+    # Delete case from loqusdb
+    LOG.info("This will delete all variants in LoqusDB for case: %s", case_id)
+    click.confirm("Do you want to continue?", abort=True)
+    LOG.info("Delete variants from LoqusDB for case: %s", case_id)
+    loqusdb_api.delete(case_id=case_id)
+
+    # If a case is deleted reset StatusDB loqusdb_id to None
```

Comment:
```suggestion
```
Code is clear enough and one could always read the doc string. It is hard to remember to update comments when the code change so they should be used with caution and when they convey context not evident in the code